src/api/store/event.py

```diff
@@ -6,9 +6,25 @@
 from .utils import get_user_or_die
 import datetime
 from datetime import timedelta
+from database.utils.constants import SHORT_STR_LEN
 import calendar
 import pytz
 from typing import Tuple
+from .utils import get_new_title
+
+def _copy_title(old_title):
+  # return first unique title
+  # Community will be None at first
+  new_title = get_new_title(None, old_title)
+  version = 0
+  while version<1000:
+    suffix = "-Copy%d" % version
+    newlen = min(len(new_title), SHORT_STR_LEN - len(suffix))
+    title = new_title[0:newlen] + suffix
+    if not Event.objects.filter(name=title).first():
+      return title
+    version += 1  
```

Comment:
Same comments as here as above.
Also, something like this should always go with tests to accompany.

Reply:
I didn't add tests, but the unit testing does check the copy routes.

Reply:
also it seems the github tests action is failing.  The pillow version in requirements.txt is not correct i suppose.  Could you update it so the tests can run ?

Reply:
All tests work now